src/Form/Checkbox/Checkbox.component.tsx

```diff
@@ -0,0 +1,123 @@
+// REACT
+import * as React from 'react';
+const { forwardRef } = React;
+
+// VENDOR
+import styled, { css } from '@xstyled/styled-components';
+import { compose } from '@xstyled/system';
+import classnames from 'classnames';
+
+// ANCHOR
+import { Check } from '../..';
+import { StyledIcon } from '../../Icon/utils';
+import {
+    space as spaceStyles,
+    SpaceProps,
+    color as colorStyles,
+    ColorProps,
+    backgroundColor as backgroundColorStyles,
+    BackgroundColorProps,
+} from '../../theme/system.theme';
+
+const HiddenInput = styled('input')<CheckboxProps>`
+    visibility: hidden;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+`;
+
+const StyledCheckbox = styled('label')<CheckboxProps>`
+    box-sizing: border-box;
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    user-select: none;
+    border: dark;
+    border-radius: 2px;
```

Comment:
I'm guessing we don't want to codify this as a `small`?

Reply:
Maybe we want to. Not sure how snowflake it is

Reply:
Let's leave this as is, but can you add a TODO calling this out?